lib/fileutils/version.rb

```diff
@@ -0,0 +1,3 @@
+module FileUtils
+  VERSION = "0.7.3-dev"
```

Comment:
Can you update this version to "0.7.3.beta1" ?

Reply:
Sure, changed.